BREAKING-CHANGES.md

```diff
@@ -0,0 +1,190 @@
+# Breaking Changes
+
+This document contains a list of breaking changes in each version and some notes to help migrate
+between versions.
```

Comment:
Maybe we can also add a link here to the [`breaking change`](https://github.com/ratatui-org/ratatui/labels/breaking%20change) label?

Reply:
done